tools/uf2/README.md

```diff
@@ -21,7 +25,7 @@ pub fn build(b: *Build) void {
 
 ```
 
-# Manually Executing elf2uf2
+Or execute elf2uf2 manually:
```

Comment:
nit: This is a subheading of `...in build.zig` which doesn't make sense. I'd either drop the header for that line, or make this one an h2 as well.